core/variant/type_info.h

```diff
@@ -43,14 +43,10 @@ struct EnableIf<false, T> {
 };
 
 template <typename, typename>
-struct TypesAreSame {
-	static bool const value = false;
-};
+inline constexpr bool typesAreSame = false;
```

Comment:
Id not name it this, it breaks all our naming standards, I'd just have it be called as originally

Reply:
What is the preferred naming scheme for `inline constexpr` variable templates?

Using `TypesAreSame` suggests it's a type, which is not.

I can see `is_vector_type` in `variant_converters.h`. There is also `_class_is_enabled` in `object.h`.

Should I use `snake_case`?

Reply:
Id say snake case indeed, as this naming isn't used anywhere else AFAIK

Reply:
Renamed to `types_are_same_v`